example-app/SantokuApp/src/fixtures/msw/datas/eventData.ts

```diff
@@ -4,6 +4,11 @@ import {partnerAccount} from './accountMinData';
 import {db} from '../db';
 
 export const eventData = () => {
+  const date = new Date();
+  // 日付を1年後に設定する
+  date.setFullYear(date.getFullYear() + 1);
```

Comment:
このファイルを見た人が混乱しないように、1年後を指定する理由をコメントに残しておきましょう。

```suggestion
  // 常に募集中のイベントが用意するため、日付を1年後に設定する
  date.setFullYear(date.getFullYear() + 1);
```

Reply:
対応commit: https://github.com/ws-4020/mobile-app-crib-notes/pull/1256/commits/8b89252303ab1d0c66ef3b78fed030e16763c2db